src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -127,6 +127,7 @@ internal IOperation CreateInternal(BoundNode boundNode)
                 case BoundKind.ArrayInitialization:
                     return CreateBoundArrayInitializationOperation((BoundArrayInitialization)boundNode);
                 case BoundKind.DefaultExpression:
```

Comment:
>case BoundKind.DefaultExpression [](start = 16, length = 32)

Should we also handle DefaultLiteral here?

Reply:
I don't think it can get here. All default literals should have either been converted or forced to natural type by initial binding, so we should have none left.

---
In reply to: [315434561](https://github.com/dotnet/roslyn/pull/37596#discussion_r315434561) [](ancestors = 315434561)